README.md

```diff
@@ -198,4 +198,32 @@ Run ```whoami``` to find out username of the current user when this command is i
 
 ```
 ansible-playbook playbook.yml -i inventory.ini
+```
+
+**Module 5**
+
+```
+poetry add gunicorn
+```
+
+```
```

Comment:
Your markdown (.md file format) is slightly off here. The triple backtick is to denote a code block